package.json

```
@@ -7,7 +7,7 @@
     "url": "git+https://github.com/h-be/acorn.git"
   },
   "scripts": {
-    "install-deps": "npm install && npm run dna-install-deps && npm run web-install && npm run electron-install",
+    "install-deps": "npm install && npm run dna-install && npm run web-install && npm run electron-install",
```

```suggestion
    "install-deps": "npm install && npm run dna-install && npm run web-install && npm run electron-install && npm run electron-tsc",
```

this would compile the typescript on first 'install' run, thus `npm run dev` would work right, when you run it first time